app/src/components/widgets/multiLevelWidgets/cumulativeTrendChart/cumulativeTrendChart.jsx

```diff
@@ -164,9 +202,16 @@ export class CumulativeTrendChart extends PureComponent {
 
   drillDown = () => this.updateActiveAttributes(this.fetchWidgetWithActiveAttributes);
 
-  showFilter = () => this.updateActiveAttributes(this.showDetailsView);
+  showFilter = () => this.updateActiveAttributes(this.navigateToTestListView);
 
-  userSettingsChangeHandler = (data) => this.props.onChangeUserSettings(data, this.getConfig);
+  userSettingsChangeHandler = (data) => {
+    if (data.defectTypes) {
+      this.setState({
```

Comment:
I think it is redundant to hold this flag in `state`.
You can get it directly from `props` where you need.